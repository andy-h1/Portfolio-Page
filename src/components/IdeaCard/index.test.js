import React from 'react';
import { render } from '@testing-library/react';
import { IdeaCard } from '.';

describe('Form Component', () => {
  it('should render', () => {
    const { container } = render(<IdeaCard />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
