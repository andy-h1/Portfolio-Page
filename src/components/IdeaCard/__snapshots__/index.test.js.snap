// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component should render 1`] = `
<div>
  <h3>
    Title:
  </h3>
  <p>
    Description: 
  </p>
</div>
`;
